src/world_interface/kinematic_common_interface.cpp

```diff
@@ -170,9 +170,23 @@ double getJointPowerValue(types::jointid_t joint) {
 
 void setJointMotorPower(robot::types::jointid_t joint, double power) {
 	using robot::types::jointid_t;
-
 	if (jointMotorMap.find(joint) != jointMotorMap.end()) {
-		setMotorPower(jointMotorMap.at(joint), power);
+		// TODO: check if associated motor is in IK_MOTORS, don't check specifically these ones
+		if (joint == jointid_t::elbow || joint == jointid_t::shoulder) {
+			if (!Globals::armIKEnabled) {
+				setMotorPower(jointMotorMap.at(joint), power);
+			}
+		} else {
+			setMotorPower(jointMotorMap.at(joint), power);
+		}
+	} else if (joint == jointid_t::ikForward || joint == jointid_t::ikUp) {
+		if (Globals::armIKEnabled) {
+			if (joint == jointid_t::ikForward) {
+				Globals::planarArmController.set_x_vel(dataclock::now(), power);
+			} else {
+				Globals::planarArmController.set_y_vel(dataclock::now(), power);
```

Comment:
I realized we should also be scaling these. Add a constant in `Constants.h` called `MAX_EE_VEL` or something that is max EE velocity in m/s. I think 0.3m/s or something is good enough, though we can tune that.

Then just scale power by this max vel, so something like:
```c++
Globals::planarArmController.set_x_vel(dataclock::now(), power * Constants::arm::MAX_EE_VEL);
```
and the same for the y vel too.